README.md

```diff
@@ -4,19 +4,33 @@
 
 [![npm](https://img.shields.io/npm/v/rdme)](https://npm.im/rdme) [![Build](https://github.com/readmeio/rdme/workflows/CI/badge.svg)](https://github.com/readmeio/rdme)
 
-`rdme` is the CLI wrapper for [ReadMe's RESTful API](https://docs.readme.com/reference/intro-to-the-readme-api). It allows you to upload and edit [OpenAPI](https://swagger.io/specification/) and [Swagger](https://swagger.io/specification/v2/) files associated with projects you create on [ReadMe](https://readme.com/). Additionally, you can sync documentation with your project, and manage project versions.
+`rdme` is [ReadMe](https://readme.com)'s official command-line interface (CLI) and [GitHub Action](#github-actions) wrapper. It allows you to sync [OpenAPI](https://spec.openapis.org) and [Swagger](https://swagger.io/specification/v2/) definitions with projects you create on [ReadMe](https://readme.com/). You can also access other parts of [ReadMe's RESTful API](https://docs.readme.com/reference/intro-to-the-readme-api), including syncing Markdown documentation with your project and managing project versions.
```

Comment:
Is https://spec.openapis.org the best place to link to for this? It looks bare and pretty ugly:

<img width="999" alt="image" src="https://user-images.githubusercontent.com/848223/155614931-fec6c7f0-c445-48a8-a9cd-39f948668d67.png">

Reply:
Ugh yeah so I took [this link](https://spec.openapis.org/oas/v3.1.0.html) and dropped most of the params so people could discover their own spec version, but agreed, that landing page is hideous and not very intuitive.

I think it was Jon who discovered that site recently and I think I like linking to [this](https://spec.openapis.org/oas/v3.1.0.html) as opposed to the giant Markdown file... maybe I'll just include the full spec file link here instead of that ugly landing page? Any thoughts on what this link should be @readmeio/api?

Reply:
I like that site you linked @kanadgupta